packages/docs/src/page-configs/ui-elements/badge/examples/WithOtherComponents.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <div style="display: flex; align-items: center;">
+    <va-badge class="mr-4" text="New" overlap>
+      <va-card>
+        <va-card-content>
+          Next update is right behind the corner
+        </va-card-content>
+      </va-card>
+    </va-badge>
+
+    <va-badge
+      text="3"
+      overlap
+      class="mr-4"
+    >
+      <va-avatar src="https://randomuser.me/api/portraits/women/5.jpg" />
+    </va-badge>
+
+    <va-badge
+      class="mr-4"
+      dot
+    >
+      <va-avatar
+        square
+        src="https://randomuser.me/api/portraits/women/8.jpg"
+      />
+    </va-badge>
+
+    <va-badge
+      text="5 notifications"
+      overlap
+    >
+      <va-button preset="primary">Profile</va-button>
+    </va-badge>
```

Comment:
<img width="174" alt="image" src="https://user-images.githubusercontent.com/29167241/205918311-ef4f2ad6-5809-426e-b756-7b47d8027bdf.png">
Lets replace this with something like this:
<img width="133" alt="image" src="https://user-images.githubusercontent.com/29167241/205919494-7457b547-91ff-4e5a-a166-e22e1bdd81c1.png">